client/layout/guided-tours/tours/simple-payments-end-of-year-guide.js

```diff
@@ -0,0 +1,173 @@
+/**
+ * External dependencies
+ *
+ * @format
+ */
+
+import React from 'react';
+import { translate } from 'i18n-calypso';
+import { overEvery as and } from 'lodash';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal dependencies
+ */
+import {
+	makeTour,
+	Continue,
+	Tour,
+	Step,
+	ButtonRow,
+	Next,
+	Quit,
+	Link,
+} from 'layout/guided-tours/config-elements';
+import { hasSelectedSitePremiumOrBusinessPlan } from 'state/ui/guided-tours/contexts';
+import { isDesktop } from 'lib/viewport';
+
+export const SimplePaymentsEndOfYearGuide = makeTour(
+	<Tour
+		name="simplePaymentsEndOfYearGuide"
+		version="20171103"
+		path={ [
+			'/stats',
+			'/plans',
+			'/pages',
+			'/posts',
+			'/media',
+			'/comments',
+			'/types',
+			'/themes',
+			'/sharing',
+			'/people',
+			'/plugins',
+			'/settings',
+		] }
+		when={ and( isDesktop, hasSelectedSitePremiumOrBusinessPlan ) }
+	>
+		<Step name="init" placement="right">
+			<p>
+				{ translate(
+					'Prepare for holiday shopping and end-of-year donations: ' +
+						'add a {{strong}}payment button{{/strong}} to sell products and services or accept donations on your site!',
+					{
+						components: {
+							strong: <strong />,
+						},
+					}
+				) }
+			</p>
+			<div style={ { textAlign: 'center' } }>
+				<img
+					src="/calypso/images/illustrations/illustration-shopping-bags.svg"
+					style={ { width: '210px', marginBottom: '10px' } }
+				/>
+			</div>
+			<ButtonRow>
+				<Next step="add-new-page">{ translate( 'Get started!' ) }</Next>
```

Comment:
Hi! I've found a possible matching string that has already been translated **89** times:
`translate( 'Get Started' )` *ES Score: 13*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/guided-tour-end-of-year)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).